_data/sites.json

```
@@ -4305,6 +4328,19 @@
         ]
     },
 
+    {
+        "name": "CoffeeCup",
+        "url": "https://www.coffeecup.com/contact/",
+        "difficulty": "hard",
+        "notes": "You need to contact them under helpme@coffeecup.com. They should reply quite fast.",
```

```suggestion
        "notes": "You need to contact them via email. They usually reply quickly.",
```
We don't add the e-mail address on the notes to keep it short, like I stated on the other PR.